### ISSUE TITLE
 \Description for images (ACM submission) are rendered

### ISSUE THREAD

**Issue by WoMeijerPhD** [arXiv/html_feedback#1742]
### Description

For ACM (SIGCHI) submissions, it is necessary to include a \Description tag for Figures, however these are rendered in the body text of the HTML version. Removing these tags manually would complicate versioning.My suggestion would be to use \Description to populate image alt text

### (Required) What is this issue most closely related to? Select one.

Figures

### Internal issue ID

947cd522-2dce-47e2-895f-7f9d0e0737bc

### Paper URL

https://arxiv.org/html/2407.12407v1

### Browser

Chrome/127.0.0.0

### Device Type

Desktop

**Comment by dginev** [comment]
Bookkeeping comment: This is pending the upgrades from https://github.com/brucemiller/LaTeXML/pull/2666

**Comment by dginev** [comment]
Thank you for your report - the specific issue is now solved.

We have regenerated the article and you should be able to see an improvement.
